packages/riipen-ui/src/components/TableColumn.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+class TableColumn extends React.Component {
```

Comment:
I am confused. Does this component not render anything? Didn't even think this was possible.

Reply:
Yeah, You can render nothing by returning null which is the default when not implemented. I am not opposed to the idea of using a node to store information used by other components to render something as that is essentially what a HOC is. But that is my opinion on this, Id love to get other people feedback. 
This allows us to pass the rendering data in the children of the Table component rather than as an additional prop. Both are valid but I prefer this way.

Reply:
I think this is fine.